src/vmm/src/persist.rs

```diff
@@ -284,32 +384,35 @@ mod tests {
         let err = InvalidVersion;
         let _ = format!("{}{:?}", err, err);
 
+        let err = InvalidVmState(vstate::Error::NotEnoughMemorySlots);
```

Comment:
Yup, it's similar to other tests that check `Display` implementations, only for coverage.

Reply:
It also tests that `Display` and `Debug` **are** implemented. It doesn't verify message string, but it does verify that there **is** a `Display` and `Debug` implementation.

Reply:
Ok, sounds good, it could be a good idea to add a test also for `LoadSnapshotError`. I noticed that the coverage decreased and I think (not 100% sure though) there isn't a lot of new functionality that can't be rust tested (either with unit tests or with integration tests).

Reply:
done